cucoslib/dispatcher/flows/monitorUpstreamFlow.yml

```diff
@@ -0,0 +1,24 @@
+---
+  flow-definitions:
+    - name: 'monitorUpstreamFlow'
+      queue: '{DEPLOYMENT_PREFIX}_ingestion_monitorUpstreamFlow_v0'
+      propagate_node_args: 'monitorUpstreamAnalysisFlow'
+      node_args_from_first: true
+      sampling:
+        name: 'constant'
+        args:
+          retry: 10
+      edges:
+        - from:
+          to: 'InitMonitoringFlow'
+        - from: 'InitMonitoringFlow'
+          to: 'monitorUpstreamAnalysisFlow'
+          condition: &shouldScheduleMonitoringCheck
+            name: 'fieldExist'
+            args:
+              key: 'document_id'
+        - from: 'monitorUpstreamAnalysisFlow'
+          to: 'PackageFinalizeTask'
+      failures:
+        - nodes: 'monitorUpstreamAnalysisFlow'
+          fallback: 'PackageFinalizeTaskError'
```

Comment:
The package analysis results seem to not be stored on S3 (`S3PackageData`).
Shouldn't there be an edge to some *PackageResultCollector* here ?